mlir/utils/jenkins/Jenkinsfile

```diff
@@ -264,18 +264,6 @@ pipeline {
                                 }
                             }
                         }
-                        stage("Tune MLIR Kernels") {
-                            when { allOf{
-                                equals expected: true, actual: params.nightly;
-                                equals expected: true, actual: params.perfTest;
-                            } }
-                            steps {
-                                buildProject('ci-performance-scripts', '')
-                                dir('build') {
-                                    sh 'python3 ./bin/MIOpenDriver.py -tuning'
-                                }
-                            }
-                        }
```

Comment:
Does this still need to exist under weekly context? Or else which stage do the tuning now?

Reply:
It is now done in the "Test MIOpen config with tuning" stage.

Reply:
Could you amend that stage name to this one:

Now that stage isn't a test any more but more precisely the `Tune MLIR Kernels` work.